### ISSUE TITLE
The motivation

### ISSUE THREAD

**Issue by Germany321** [om-ai-lab/VLM-R1#10]
Can your model generate COT just like Deepseek-R1? If not, I am curious what is the motivation to use RL for training?

**Comment by Germany321** [comment]
I just looked through the task in the repo. The authors focus on single-object grounding. This task does not have a long language response.

**Comment by xrc10** [comment]
The model can generate COT. And that is the motivation of using GRPO because the training data does not have ground truth COT annotation. Please check out the demo here: https://huggingface.co/spaces/omlab/VLM-R1-Referral-Expression

**Comment by Germany321** [comment]
Thanks for your reply. Are SFT results trained on COT data? Or you just supervise the model with the IOU + format reward and let the model learn to generate COT by themselves?

**Comment by snakeztc** [comment]
There is no COT SFT involved. We just do GPRO on the Owen base model directly and the reasoning process is emerged automatically.

**Comment by CaptainEven** [comment]
> There is no COT SFT involved. We just do GPRO on the Qwen-2.5 VL 3B directly and the reasoning process is emerged automatically.

Excellent!

**Comment by Germany321** [comment]
> There is no COT SFT involved. We just do GPRO on the Qwen-2.5 VL 3B directly and the reasoning process is emerged automatically.

That sounds very interesting!

**Comment by chenyangzhu1** [comment]
> There is no COT SFT involved. We just do GPRO on the Qwen-2.5 VL 3B directly and the reasoning process is emerged automatically.不涉及 COT SFT。我们只需直接在 Qwen-2.5 VL 3B 上进行 GPRO，推理过程就会自动出现。

Very impressive, but I am curious whether this naturally emergent reasoning process itself is correct and whether it can help the model obtain more accurate answers, or if it is merely content generated by the model to achieve higher format rewards.

**Comment by snakeztc** [comment]
actually the format reward is easily achieved almost perfect reward. So most of the later improvement in on the IoU reward
